docs/eventing/sink/README.md

```diff
@@ -0,0 +1,46 @@
+A Sink is an Addressable resource that acts as a link
+between the Eventing mesh and an entity or system.
+
+We can connect any `Source` to a `Sink`, like `PingSource` and `KafkaSink`:
```

Comment:
Since sink is a concept, I'd keep it as "sink" throughout the doc, same for "source".
`PingSource` and `KafkaSink` are fine but I'd maybe say "such as `PingSource` or `KafkaSink` objects".